cpp/modules/deck.gl/core/src/lib/deck.h

```diff
@@ -72,8 +72,8 @@ class Deck : public Component {
   int height{100};  // Dummy value, ensure something is visible if user forgets to set window size
 
   std::shared_ptr<lumagl::AnimationLoop> animationLoop{new lumagl::GLFWAnimationLoop()};
-  std::shared_ptr<LayerContext> context{new LayerContext{this, this->animationLoop->device()}};
   std::shared_ptr<ViewManager> viewManager{new ViewManager()};
+  std::shared_ptr<LayerContext> context{new LayerContext{this, this->animationLoop->device()}};
```

Comment:
I wonder if we should call this `layerContext` here in Deck? context is kind of an evil word as it can be used in many places where some kind of non-local plumbing is needed. Here we can see the type, but when we use it later it is less obvious